spring-integration-webflux/src/main/java/org/springframework/integration/webflux/outbound/WebFluxRequestExecutingMessageHandler.java

```diff
@@ -393,7 +401,8 @@ private Object createReplyFromResponse(Mono<ResponseEntity<Flux<Object>>> respon
 				.map(this::getReply);
 	}
 
-	private Map<String,Object> evaluateAttributeVariables(Message<?> requestMessage){
+	@SuppressWarnings("unchecked")
+	private Map<String, Object> evaluateAttributeVariables(Message<?> requestMessage) {
 		return this.attributeVariablesExpression.getValue(this.evaluationContext, requestMessage, Map.class);
 	}
```

Comment:
You haven't addressed this code style concern: black lines around class member.

Reply:
I didnt quite catch this one. I did reformat from code style but it didnt make any difference. Do you mean adding a blank line before and after return statement like below?
```
@SuppressWarnings("unchecked")
	private Map<String, Object> evaluateAttributeVariables(Message<?> requestMessage) {
		
		return this.attributeVariablesExpression.getValue(this.evaluationContext, requestMessage, Map.class);

	}
```

Reply:
No, the `return` statement is not class member.
The `evaluateAttributeVariables()` method is the one.
So, exactly this method has to be surrounded with blank lines.